src/measure_theory/integral/circle_integral.lean

```diff
@@ -0,0 +1,513 @@
+/-
+Copyright (c) 2021 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import measure_theory.integral.interval_integral
+import analysis.normed_space.pointwise
+import analysis.special_functions.non_integrable
+import analysis.analytic.basic
+
+/-!
+# Integral over a circle in `ℂ`
+
+In this file we define `∮ z in C(c, R), f z` to be the integral $\oint_{|z-c|=|R|} f(z)\,dz$ and
+prove some properties of this integral.
+
+## Main definitions
+
+* `circle_map c R`: the exponential map $θ ↦ c + R e^{θi}$;
+
+* `circle_integrable f c R`: a function `f : ℂ → E` is integrable on the circle with center `c` and
+  radius `R` if `f ∘ circle_map c R` is integrable on `[0, 2π]`;
+
+* `circle_integral f c R`: the integral $\oint_{|z-c|=|R|} f(z)\,dz$, defined as
+  $\int_{0}^{2π}(c + Re^{θ i})' f(c+Re^{θ i})\,dθ$;
+
+* `cauchy_power_series f c R`: the power series that depends only on `f ∘ circle_map c R` and
+  converges to `f w` if `f` is differentiable on the closed ball `metric.closed_ball c R`
+  and `w` belongs to the corresponding open ball.
```

Comment:
I think this description is cryptic. The minimal improvement would be to write "ℂ-differentiable", but I think you can write the coefficient in front of `z^n` is `(2 * π * I : ℂ)⁻¹ • ∮ z in C(c, R), (z - c)⁻¹ ^ n • (z - c)⁻¹ • f z` here (it's not even strictly necessary to explain the notation).